lib/handler/pool/state.ex

```diff
@@ -0,0 +1,94 @@
+defmodule Handler.Pool.State do
+  @moduledoc false
```

Comment:
I think some docs here would be helpful :)

Reply:
👍 I will definitley be adding docs, but I want to make sure I have the interface nailed down before I start writing them up. I'm guessing I will have at least one thing that I have to re-work in a big way before this is ready to put into peg